live-build/misc/ansible-roles/appliance-build.devops-development/tasks/main.yml

```diff
@@ -15,18 +15,10 @@
 #
 
 ---
-#
-# We are wrapping this in a retry block because the Gitlab server sometimes
-# hangs up unexpectedly. We haven't seen this with GitHub, so we should
-# probably remove the retry once we switch to GitHub.
-#
 - git:
-    repo: 'https://gitlab.delphix.com/devops/devops-gate.git'
-    version: master
+    repo: "https://{{ lookup('env', 'GITHUB_TOKEN') }}@github.com/devops-gate/zfs.git"
```

Comment:
you're changing the devops URL to the zfs URL?

Reply:
I think this URL should be `github.com/delphix/devops-gate.git`?

Reply:
Fixed, thank you :joy: